modules/web/src/app/node-data/component.ts

```diff
@@ -130,12 +135,39 @@ export class NodeDataComponent extends BaseFormValidator implements OnInit, OnDe
     private readonly _settingsService: SettingsService,
     private readonly _osmService: OperatingSystemManagerService,
     private readonly _projectService: ProjectService,
+    private readonly _quotaCalculationService: QuotaCalculationService,
     private readonly _params: ParamsService,
     private readonly _cdr: ChangeDetectorRef
   ) {
     super();
   }
 
+  @ViewChild('quotaWidgetContainer', {read: ViewContainerRef})
+  set quotaWidgetContainer(ref: ViewContainerRef) {
+    if (ref && this.isEnterpriseEdition && !this.quotaWidgetComponentRef) {
+      this._quotaCalculationService
+        .getQuotaCalculations(this.projectId, this.provider)
+        .pipe(takeUntil(this._unsubscribe))
+        .subscribe((calculatedQuota: ResourceQuotaUpdateCalculation) => {
+          this.quotaWidgetComponentRef.updateEstimatedQuota(calculatedQuota);
+        });
+
+      this.quotaWidgetComponentRef = ref.createComponent(QuotaWidgetComponent).instance;
+      this.quotaWidgetComponentRef.projectId = this.projectId;
+      this.quotaWidgetComponentRef.showQuotaWidgetDetails = true;
+      this.quotaWidgetComponentRef.showIcon = true;
+      this.quotaWidgetComponentRef.estimatedQuotaExceeded
+        .pipe(takeUntil(this._unsubscribe))
+        .subscribe((quotaExceeded: boolean) => {
+          if (quotaExceeded) {
+            this.form.get(Controls.Count).setErrors({quotaExceeded: true});
```

Comment:
I think we should set this error on `this.form` instead of `Count` control because in some cases quota might get exceed due to some other parameters (disk size, memory etc.).